test/conformance/channel_tracing_test.go

```diff
@@ -25,17 +25,19 @@ import (
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 	"knative.dev/eventing/test/conformance/helpers"
 	testlib "knative.dev/eventing/test/lib"
-	"knative.dev/eventing/test/lib/resources"
 
 	contribtest "knative.dev/eventing-kafka/test"
 )
 
+// Eventing channels are v1, but Kafka channel is v1beta1.
+const kafkaChannelAPIVersion = "messaging.knative.dev/v1beta1"
```

Comment:
Can we make use of this, for exactly the reasons you added a comment, here:

https://github.com/knative-sandbox/eventing-kafka/blob/d8d8d2333cd7735c7dd538825ea6a5ed4b2bf9f3/test/upgrade/test_main.go#L34

:pray:

Reply:
related: 

* https://github.com/knative-sandbox/eventing-kafka/pull/467/files#r605554669
* https://github.com/knative/eventing/pull/5201/#pullrequestreview-629025730